_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,271 @@
+
+```html
+<style>
+	p {
+		letter-spacing: 0.1em !important;
+	}
+</style>
+
+<p style="letter-spacing: 0.15em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 7
+
+Both this `p` and `span` elements match the **not important** condition. For the `span`, the [cascaded][] value is `inherit !important`, thus the [computed][] value is the [inherited][] value, that is the [computed][] value of its parent, and it is not [important][]. Note that neither the **above minimum** (because the [computed][] value is only 0.1 times the font size), nor the **cascade** (because the [cascaded][] value comes from the `style` attribute) conditions are matched. Even though the value is too small, styles with [author origin][] declared by assistive technologies may win the [cascade sort][] and override it, thus this may satisfy [Success Criterion 1.4.12 Text Spacing][sc1412] and does not fail this rule. This is nonetheless bad practice and sufficient spacing should be used.
+
+```html
+<p style="letter-spacing: 0.1em">
+	<span style="letter-spacing: inherit !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 8
+
+Both this `p` and `span` elements match the **not important** condition. For the `span`, the [cascaded][] value is `unset !important`, which is equivalent as `inherit` since it is an inherited property, thus the [computed][] value is the [inherited][] value, that is the [computed][] value of its parent, and it is not [important][]. Even though the value is too small, styles with [author origin][] declared by assistive technologies may win the [cascade sort][] and override it, thus this may satisfy [Success Criterion 1.4.12 Text Spacing][sc1412] and does not fail this rule. This is nonetheless bad practice and sufficient spacing should be used.
+
+```html
+<p style="letter-spacing: 0.1em">
+	<span style="letter-spacing: unset !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `p` element has a [computed][] `letter-spacing` of `1.6px` (assuming a default `medium` font size of `16px`) which is below the recommended minimum.
```

Comment:
```suggestion
This `p` element has a [computed][] `letter-spacing` of 0.1 times the fonts which is below the recommended minimum.
```

Reply:
I debated than one with myself for quite some times before deciding to go that way 🤔 

My reasoning is that the Expectation is about the computed style, so the description should also be. Which I agree is annoying in forcing to assume the default value even though the example would fail anyway…